bundle/config/mutator/translate_paths.go

```diff
@@ -47,13 +47,22 @@ func (err ErrIsNotNotebook) Error() string {
 	return fmt.Sprintf("file at %s is not a notebook", err.path)
 }
 
-type translatePaths struct{}
+type translatePaths struct {
+	dashboardsOnly bool
+}
 
 // TranslatePaths converts paths to local notebook files into paths in the workspace file system.
 func TranslatePaths() bundle.Mutator {
 	return &translatePaths{}
 }
 
+// TranslatePathsDashboards converts paths to local dashboard files into paths in the workspace file system.
+func TranslatePathsDashboards() bundle.Mutator {
+	return &translatePaths{
+		dashboardsOnly: true,
```

Comment:
Given that you already have a separate mutator, why not just implement Apply for it directly and keep translatePaths option-less.

Reply:
I tried that, but i do not want to copy translation logic (structs and methods) nor i want to make it public to other packages, so i decided this is the "cleanest" way to get this done

Reply:
Hrm, I expected it to be simpler than that. We can proceed with either, up to you.